src/templates/default.json

```diff
@@ -1,35 +1,34 @@
 {
-  "title": "Default",
-  "name": "default",
+  "title": "Odyssey Template",
+  "name": "odyssey-template",
   "version": "2.0.0",
-  "description": "Provides a simple User authentication system.",
+  "description": "Provides the default Odyssey backbone for all US states.",
+  "preview": {
+    "url": "https://formio.github.io/formio-app-formio/dist",
+    "repo": "https://github.com/formio/formio-app-formio"
+  },
   "roles": {
     "administrator": {
       "title": "Administrator",
       "description": "A role for Administrative Users.",
-      "admin": true,
-      "default": false
+      "admin": true
     },
-    "authenticated": {
-      "title": "Authenticated",
-      "description": "A role for Authenticated Users.",
-      "admin": false,
-      "default": false
+    "guardian": {
+      "title": "Guardian",
+      "description": "A role for Guardian Users."
     },
     "anonymous": {
       "title": "Anonymous",
       "description": "A role for Anonymous Users.",
-      "admin": false,
```

Comment:
does removing this line mean that we want anonymous users to be admins? Not sure I follow here.

Reply:
I think the default is false IIRC